spec/controllers/reservations_controller_spec.rb

```diff
@@ -1656,6 +1604,10 @@
   end
 
   describe '#deny_request PUT' do
+    # SMELL: this doesn't belong in the controller
```

Comment:
Probably, but both of these methods are definitely leaving the controller so I don't think these tests need that much attention.